test/app.test.js

```diff
@@ -610,10 +610,11 @@ describe('app', function() {
   });
 
   describe('app.model(Model)', function() {
-    var app, db;
+    var app, db, MyTestModel;
     beforeEach(function() {
       app = loopback();
       db = loopback.createDataSource({ connector: loopback.Memory });
+      MyTestModel = app.registry.createModel('MyTestModel');
```

Comment:
I take my comment back. I see that this model is used in several test, in which case it's best to reuse it.